Modules/System/Data Compression/src/DataCompressionImpl.Codeunit.al

```diff
@@ -108,6 +108,27 @@ codeunit 421 "Data Compression Impl."
         end;
     end;
 
+    procedure IsZip(InputInStream: InStream): Boolean
+    var
+        impl2: Codeunit "Data Compression Impl.";
```

Comment:
CodeCop will reject this variable name. Must be DataCompressionImpl2, I fear.

Reply:
On a second thought, I agree with @pri-kise on this. I personally try to avoid a `TryFunction` for business logic, especially if "low level" like this. I have rewritten the check and now it's testing the ZIP header - just as `IsGZip` does.